contracts/src/core/interfaces/IEigenDADirectory.sol

```diff
@@ -48,72 +50,39 @@ interface IEigenDAConfigRegistry {
     error ConfigAlreadyExists(string name);
     error ConfigDoesNotExist(string name);
 
-    event ConfigBytes32Added(string name, bytes32 indexed key, bytes32 value, string extraInfo);
-    event ConfigBytesAdded(string name, bytes32 indexed key, bytes value, string extraInfo);
-    event ConfigBytes32Replaced(string name, bytes32 indexed key, bytes32 oldValue, bytes32 newValue, string extraInfo);
-    event ConfigBytesReplaced(string name, bytes32 indexed key, bytes oldValue, bytes newValue, string extraInfo);
-    event ConfigBytes32Removed(string name, bytes32 indexed key);
-    event ConfigBytesRemoved(string name, bytes32 indexed key);
```

Comment:
getting rid of events b/c everything is onchain now? Might still be useful to keep in case someone wants to write a subgraph to index the values? Not sure..

Reply:
No, the events are just a lot less now because there is only additions to the configs allowed. They are defined in the library (since this allows events to be tested from the library in isolation which is a TODO)